src/test/kotlin/no/nav/helse/spion/db/integration/DatabaseTests.kt

```diff
@@ -0,0 +1,134 @@
+package no.nav.helse.spion.db.integration
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.zaxxer.hikari.HikariConfig
+import com.zaxxer.hikari.HikariDataSource
+import io.ktor.config.ApplicationConfig
+import no.nav.helse.YtelsesperiodeGenerator
+import no.nav.helse.spion.db.PostgresYtelsesperiodeDao
+import no.nav.helse.spion.db.YtelsesperiodeDao
+import no.nav.helse.spion.db.createLocalHikariConfig
+import no.nav.helse.spion.domene.Arbeidsgiver
+import no.nav.helse.spion.domene.Periode
+import no.nav.helse.spion.domene.Person
+import no.nav.helse.spion.domene.ytelsesperiode.Arbeidsforhold
+import no.nav.helse.spion.domene.ytelsesperiode.repository.PostgresRepository
+import no.nav.helse.spion.domene.ytelsesperiode.Ytelsesperiode
+import org.apache.commons.lang3.time.StopWatch
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.time.LocalDate
+import java.util.concurrent.TimeUnit
+import kotlin.system.measureTimeMillis
+import kotlin.test.assertEquals
+
+class postgresTests {
+
+
+    val testYtelsesPeriode = Ytelsesperiode(
+            periode = Periode(LocalDate.of(2019, 1, 1), LocalDate.of(2019, 2, 1)),
+            arbeidsforhold = Arbeidsforhold(
+                    arbeidsforholdId = "1",
+                    arbeidstaker = Person("Solan", "Gundersen", "10987654321"),
+                    arbeidsgiver = Arbeidsgiver("Flåklypa Verksted", "666666666", "555555555", null)
+            ),
+            vedtaksId = "1",
+            refusjonsbeløp = BigDecimal(10000),
+            status = Ytelsesperiode.Status.INNVILGET,
+            grad = BigDecimal(50),
+            dagsats = BigDecimal(200),
+            maxdato = LocalDate.of(2019, 1, 1),
+            ferieperioder = emptyList(),
+            ytelse = Ytelsesperiode.Ytelse.SP,
+            merknad = "Fritak fra AGP",
+            sistEndret = LocalDate.now()
+    )
+    val mapper = ObjectMapper().registerModule(KotlinModule()).registerModule(JavaTimeModule()).configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false).configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```

Comment:
For å slippe å ha mange steder der vi må konfigurere opp objectmappere kan du bruke Koin i testen og hente den som blir satt opp før bruk prod-delen av koden. Se SaksgangConsumerTest